ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -35,6 +36,15 @@ object Cli {
     override val reads: String => Duration = Duration.parse
   }
 
+  private implicit val clientAuthRead: Read[ClientAuth] = Read.reads {
```

Comment:
I don’t care that much about the CLI format here but this was the most reasonable option that I came up with.